### PR TITLE
Drup8 1149

### PR DESCRIPTION
**Pull request by mechanoid23**
Okay folks, I think this one is ready to go.

### CODE REVIEW

File: modules/osu_migrations_files/migrations/upgrade_d7_files.yml [hunk 1]
```diff
@@ -35,11 +35,11 @@ source:
     #
     # See the configuration for the source_full_path property in the process
     # section below.
-    source_base_path: '/var/www/html/docroot'
+    source_base_path: 'https://health.oregonstate.edu'
```
Comment (Ruefa) [inline modules/osu_migrations_files/migrations/upgrade_d7_files.yml]: Is there a way we can get this programmatically?

File: modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+id: upgrade_d7_media_audio
+#class: Drupal\osu_migrations_media\Plugin\migrate\source\FileByType
```
Comment (Ruefa) [inline modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml]: Need this comment?

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: modules/osu_migrations_files/migrations/upgrade_d7_files.yml (hunk 1 above)
Comment (emerham) [reply modules/osu_migrations_files/migrations/upgrade_d7_files.yml]: we should leave the source path as /var/www/html/docroot and only change it on a case-by-case migration.
At some point we will want to also migrate private files and those have to be available on the file system to migrate.

File: modules/osu_migrations_files/migrations/upgrade_d7_files.yml [hunk 3]
```diff
@@ -35,11 +35,11 @@ source:
...
 process:
   # If you are using this file to build a custom migration consider removing
   # the fid field to allow incremental migrations.
-  fid: fid
+#  fid: fid
```
Comment (emerham) [inline modules/osu_migrations_files/migrations/upgrade_d7_files.yml]: go ahead and remove this line completely.

File: modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml (hunk 2 above)
Comment (emerham) [reply modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml]: Class requirements in yaml are only if we need a seperate migration class:
[Drupal Migrate Yaml Class Key](https://understanddrupal.com/blog/configuration-options-migration-yaml-definition-files/#class)

Comment (emerham) [reply modules/osu_migrations_media/migrations/upgrade_d7_media_audio.yml]: With that the class key should be removed from the yamls or set to the default `\Drupal\migrate\Plugin\Migration`
I have been setting this previously to the default for out of habit.

**Review by emerham — CHANGES_REQUESTED** [review]

**Pull request closed without merging**